src/main/java/uk/gov/justice/hmpps/prison/repository/sql/InmateRepositorySql.kt

```diff
@@ -3,30 +3,31 @@ package uk.gov.justice.hmpps.prison.repository.sql
 enum class InmateRepositorySql(val sql: String) {
   FIND_INMATE_DETAIL(
     """
-        SELECT B.OFFENDER_BOOK_ID,
+        SELECT B.OFFENDER_BOOK_ID BOOKING_ID,
         B.BOOKING_NO,
         O.OFFENDER_ID,
         O.ROOT_OFFENDER_ID,
-        O.OFFENDER_ID_DISPLAY,
+        O.OFFENDER_ID_DISPLAY OFFENDER_NO,
         O.FIRST_NAME,
         CONCAT(O.middle_name, CASE WHEN middle_name_2 IS NOT NULL THEN concat(' ', O.middle_name_2) ELSE '' END) MIDDLE_NAME,
         O.LAST_NAME,
-        B.AGY_LOC_ID,
-        B.LIVING_UNIT_ID,
-        B.ACTIVE_FLAG,
+        B.AGY_LOC_ID AGENCY_ID,
+        B.LIVING_UNIT_ID ASSIGNED_LIVING_UNIT_ID,
+        CASE WHEN B.ACTIVE_FLAG = 'Y' THEN 1 ELSE 0 END ACTIVE_FLAG,
```

Comment:
could this potentially break existing clients? changing type from char to boolean? 

Or is this handle by some other layer?

Reply:
guess it was broken previously eitherway

Reply:
It used a different mapper